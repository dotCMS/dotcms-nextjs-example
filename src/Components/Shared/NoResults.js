import React from "react";

const NoResults = () => {
  return (
    <>
        <div className="alert alert-info">
            No results available.
        </div>
    </>
  );
};

export default NoResults;
