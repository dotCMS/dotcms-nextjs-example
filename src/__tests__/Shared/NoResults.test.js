import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import NoResults from '../../Components/Shared/NoResults';

describe('<NoResults />', () => {
    it('renders and matches snapshot', () => {
        const wrapper = shallow(<NoResults />);
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});
