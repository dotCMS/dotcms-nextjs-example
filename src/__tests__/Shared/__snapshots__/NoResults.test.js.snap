// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResults /> renders and matches snapshot 1`] = `
<Fragment>
  <div
    className="alert alert-info"
  >
    No results available.
  </div>
</Fragment>
`;
